### PR TITLE
Add prototype

### PR DESCRIPTION
**Pull request by d10n**

### CODE REVIEW

File: main/main.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			o.Logger = logger
+
+			cacheEnv := os.Getenv("UOR_CACHE")
+			if cacheEnv != "" {
+				o.CacheDir = cacheEnv
+			} else {
+				home, err := homedir.Dir()
+				if err != nil {
+					return err
+				}
```
Comment (jpower432) [inline main/main.go]: The client by default, creates the cache ina different location shown [here](https://github.com/uor-framework/uor-client-go/blob/79f8cf0595192625bfe33d73bbf5d6708c17ba33/cmd/client/commands/options/common.go#L48). This should be updated.

File: fs/fs.go [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+		case ocispec.MediaTypeImageManifest:
+			continue
+		}
+
+		if layerInfo.Annotations == nil {
+			fs.Logger.Debugf("layer unexpectedly had no annotations, ignoring: %v", layerInfo.Digest)
+			continue
+		}
+
+		skip := func(_ string) bool { return false }
```
Comment (jpower432) [inline fs/fs.go]: Nit: You can pass `nil` here. The function it calls checks if the `skip` function is nil before it's evaluated.

**Review by jpower432 — COMMENTED** [review]
Initial review: a few nits

**Comment by d10n** [comment]
Merging as-is. There are Emporous API changes which will be resolved later.

**Pull request merged**